tests/test_cargo_build_lib.rs

```diff
@@ -52,3 +53,16 @@ test!(build_with_no_lib {
                 execs().with_status(101)
                        .with_stderr("no library targets found"));
 });
+
+test!(build_with_relative_cargo_home_path {
+    let p = project("foo")
+        .file("Cargo.toml", &basic_bin_manifest("foo"))
+        .file("src/main.rs", r#"
+            fn main() {}
+        "#);
+        env::set_var("CARGO_HOME", "./cargo_home/");
```

Comment:
There are no assertions in this test that `cargo_home` is used and I can't think of a reason why existing tests would fail if `CARGO_HOME` were set like this.